lib/pbench/test/unit/server/test_requests.py

```diff
@@ -122,6 +125,26 @@ def verify_logs(caplog):
             assert record.levelname not in ("ERROR", "CRITICAL")
         assert caplog.records[-1].levelname == "WARNING"
 
+    @pytest.fixture(scope="function", autouse=True)
+    def fake_filetree(self, monkeypatch):
+        class FakeTarball:
+            def __init__(self, path: Path):
+                self.tarball_path = path
+
+        def fake_constructor(self, options: PbenchServerConfig, logger: Logger):
+            pass
+
+        def fake_create(self, controller: str, path: Path) -> None:
+            TestUpload.controller_created = controller
+            TestUpload.tarball_created = path
+            return FakeTarball(path)
+
+        TestUpload.controller_created = None
+        TestUpload.tarball_created = None
```

Comment:
I found it surprising that the class initialization of these attributes was insufficient...if that is indeed the case, then I would appreciate a code comment pointing that out and perhaps illuminating the reason why.